spec/lib/config_spec.rb

```diff
@@ -47,6 +60,19 @@ module Fusuma
       }
     end
 
+    let(:keymap_with_trigger_interval) do
+      {
+        'swipe' => {
+          4 => {
+            'right'  => {
+              'shortcut' => 'alt+Right',
+              'interval' => 0.3
+            },
```

Comment:
__[RuboCop]__

Style/TrailingCommaInHashLiteral: Avoid comma after the last item of a hash.


<!-- Comment by [Sider](https://sider.review) -->